util/insert.go

```diff
@@ -0,0 +1,116 @@
+package util
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/bmeg/grip/gripql"
+	log "github.com/sirupsen/logrus"
+)
+
+type graphElementArray struct {
+	vertices []*gripql.Vertex
+	edges    []*gripql.Edge
+}
+
+func newGraphElementArray(vertexBufSize, edgeBufSize int) *graphElementArray {
+	if vertexBufSize != 0 {
+		return &graphElementArray{vertices: make([]*gripql.Vertex, 0, vertexBufSize)}
+	}
+	if edgeBufSize != 0 {
+		return &graphElementArray{edges: make([]*gripql.Edge, 0, edgeBufSize)}
+	}
+	return nil
+}
+
+// SteamBatch a stream of inputs and loads them into the graph
+// This function assumes incoming stream is GraphElemnts from a single graph
+func SteamBatch(stream <-chan *gripql.GraphElement, vertexAdd func([]*gripql.Vertex) error, edgeAdd func([]*gripql.Edge) error) error {
```

Comment:
Should we consider having batch size as an argument?

Reply:
Please add logic to ensure each `GraphElement` references the correct graph and add a unit test.

Reply:
Done.